src/v0/destinations/pinterest_tag/transform.js

```diff
@@ -118,26 +115,17 @@ const process = (event) => {
       throw new InstrumentationError(`message type ${messageType} is not supported`);
   }
 
+  const endpoint = getEventsEndpoint(adAccountId);
   toSendEvents.forEach((sendEvent) => {
-    respList.push(responseBuilderSimple(sendEvent));
+    if (apiVersion === API_VERSION.v5) {
+      respList.push(responseBuilderSimpleNew(sendEvent, endpoint, conversionToken));
```

Comment:
can't we consider the `responseBuilderSimple` as an individual util function, that can be used for both v3 and v5?

Reply:
All V3 and V5 logic is moved to transformerV3 and transformerV5. Just import the functions here.